src/data/constants.ts

```diff
@@ -232,7 +232,7 @@ export const Statblock5e: StatblockItem[] = [
         type: "table",
         id: nanoid(),
         properties: ["stats"],
-        headers: ["Str", "Dex", "Con", "Wis", "Int", "Cha"],
+        headers: ["Str", "Dex", "Con", "Int", "Wis", "Cha"],
```

Comment:
I am not sure if this is a right place to fix, but this configuration differs with stats in imports, and it results incorrect display of wisdom / intelligence